Tests iOS/MyList/ArchiveTests.swift

```diff
@@ -126,7 +126,7 @@ class ArchiveTests: XCTestCase {
             fatalError("Unexpected request")
         }
 
-        app.myListView.readdSwipeButton.wait().tap()
+        app.myListView.readSwipeButton.wait().tap()
```

Comment:
This may _seem_ wrong, but it is actually "readd", which can be updated to something better, but think of it as "Re-add" (i.e "Move to My List", which is what the button is actually named now, so perhaps that's a better name).

Reply:
hahah ohh I thought it was to put it back in `read` status vs `reAdd`, but I changed it to `moveToMyListSwipeButton` to be clear. thanks for the catch!